datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestStickyMulticastStrategy.java

```diff
@@ -499,6 +501,62 @@ public void testExtraTasksAreNotAssignedDuringReassignment() {
     }
   }
 
+  @Test
+  public void testReBalancingTasksWithThreshold() {
+    String[] instances = new String[]{"instance1", "instance2", "instance3"};
+    reBalancingTasksWithThresholdHelper(instances, new int[]{9, 9, 13}, 1);
```

Comment:
Suggestion to add some examples with even number of instances and only 1 instance.